src/parser/jobs/rdm/modules/Procs.tsx

```diff
@@ -154,6 +208,31 @@ export class Procs extends CoreProcs {
 			why: this.getMissedProcWhy(missedFire, missedStone),
 		}))
 
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.GRAND_IMPACT.icon,
+			content: this.getCriticalProcContent(this.data.statuses.GRAND_IMPACT_READY.id),
+			tiers: SEVERITY_CRITICAL_PROCS,
+			value: this.getDropCountForStatus(this.data.statuses.GRAND_IMPACT_READY.id),
+			why: this.getCriticalProcWhy(this.data.statuses.GRAND_IMPACT_READY.id),
+		}))
+
+		//Current Thought is we want these to stand on their own, leaving this here for now in case minds are changed.
+		// this.suggestions.add(new TieredSuggestion({
+		// 	icon: this.data.actions.PREFULGENCE.icon,
+		// 	content: this.getCriticalProcContent(this.data.statuses.PREFULGENCE_READY.id),
+		// 	tiers: SEVERITY_CRITICAL_PROCS,
+		// 	value: this.getDropCountForStatus(this.data.statuses.PREFULGENCE_READY.id),
+		// 	why: this.getCriticalProcContent(this.data.statuses.PREFULGENCE_READY.id),
+		// }))
+
+		// this.suggestions.add(new TieredSuggestion({
+		// 	icon: this.data.actions.VICE_OF_THORNS.icon,
+		// 	content: this.getCriticalProcContent(this.data.statuses.THORNED_FLOURISH.id),
+		// 	tiers: SEVERITY_CRITICAL_PROCS,
+		// 	value: this.getDropCountForStatus(this.data.statuses.THORNED_FLOURISH.id),
+		// 	why: this.getCriticalProcContent(this.data.statuses.THORNED_FLOURISH.id),
+		// }))
+
```

Comment:
See comment above for commented code when tracking these procs.  If we do keep separated modules, remove this commented code.

Reply:
That is the plan, this code is only being kept in case they want it all under the one proc window.  However if I don't get much feed back after the release, then I'll just remove these and keep it with the setup I have implemented currently